domain/application/state/application.go

```diff
@@ -865,6 +868,8 @@ WHERE application_uuid = $applicationScale.application_uuid
 		appDetails, err := st.getApplicationDetails(ctx, tx, appName)
 		if err != nil {
 			return errors.Capture(err)
+		} else if appDetails.IsApplicationSynthetic {
+			return errors.Errorf("cannot set scaling state for synthetic application %q", appName)
```

Comment:
todo: work on this wording, an end user will have no idea why a synthetic application is.

I'd return an application not found error - with an error log message